snapcraft/tests/test_yaml.py

```diff
@@ -283,7 +283,89 @@ def test_invalid_yaml_invalid_name_as_number(self, mock_loadPlugin):
             snapcraft.yaml.Config()
 
         self.assertEqual(raised.exception.message,
-                         '1 is not of type \'string\'')
+                         'The \'name\' property does not match the required '
+                         'schema: 1 is not of type \'string\'')
```

Comment:
right. I mean the exception to the rule of using single quotes to avoid me going nuts is when the string contains single quotes. In that case, use double quotes. Hah, confusing. So, the style we are trying to follow in the code is:
"The 'name' property does not match the required "
"schema: 1 is not of type 'string'"

Of course not all the old code is following it, as your deleted lines show.